collections/_posts/2022-11-23-Multi Channel Stochastic Variational Inference.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: review
+title: "Multi-Channel Stochastic Variationnal Inference"
+tags: "Fusion, Multimodality, Machine Learning"
```

Comment:
Tags are not comma-separated, but rather separated by spaces, so they should not contain spaces themselves. They're also not delimited by quotes. See reference [here](https://github.com/creatis-myriad/creatis-myriad.github.io/blob/a39a9840dd2f7513cba4f767d357cf1fc37cd1ec/collections/_posts/2022-06-01-EmergingPropertiesSSViT.md?plain=1#L4).
```suggestion
tags: fusion multimodality machine-Learning
```